play_motion:
  controllers:
  - arm_controller
  - torso_controller

  motions:
    extend:
      joints:
      - arm_1_joint
      - arm_2_joint
      - arm_3_joint
      - arm_4_joint
      - arm_5_joint
      - arm_6_joint
      - arm_7_joint
      - torso_lift_joint
      points:
      - positions: [1.57, -1.0, -3.14, 1.0, -1.57, 0.0, 0.0, 0.15]
        time_from_start: 5.0
      meta:
        name: Extend
        usage: demo
        description: "Extend arm"
